pkg/runner/conn.go

```diff
@@ -0,0 +1,134 @@
+package runner
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"os"
+	"strings"
+
+	"go.uber.org/zap"
+)
+
+/*
+
+				   Instance model
+				    |          ^
+functionRequest     |          |   functionResponse
+				    v          |
+			     producer   consumer
+			        |           ^
+	bytes	        |           |     bytes
+					|           ----------------------|
+					|----------------------------> process
+
+*/
+
+var (
+	// split byte just for functionRequest package delimiter
+	splitByte = []byte("littledrizzle")
+)
```

Comment:
What is the usage for this delimiter variable

Reply:
I wil add note about it like '{"a": "b"}littledrizzle{"a": "c"}' in the pipe